openedx/features/survey_report/models.py

```diff
@@ -0,0 +1,29 @@
+"""
+Survey Report models.
+"""
+
+from django.db import models
+from jsonfield import JSONField
+
+
+class SurveyReport(models.Model):
+    """
+    This model stores information to automate the way of gathering impact data from the openedx project.
+
+    .. no_pii:
+    """
+    courses_offered = models.BigIntegerField()
+    learners = models.BigIntegerField()
+    registered_learners = models.BigIntegerField()
```

Comment:
What's the difference between these two?

Reply:
learners: Recently active users with login in the last 3 weeks.
registered_learners: Total number of users ever registered in the platform.

Reply:
Can we add that to the docstring? why _exactly_ 3 weeks? can that number change?

Reply:
> The value itself is a threshold just like any other. It was the product of the analisis that was used for the first survey.
> 
> However as far as I can tell the value was 4. https://gist.github.com/feanil/0b49d5219ae08baf1216556d455dfdbb#file-instruction-md. A value of 4 would also be easier to understand intuitively, it is the users that where active in the last Month.

This is the reason why, but I agree , the number must be 4.